exporter/clickhousetracesexporter/writer.go

```diff
@@ -86,10 +86,19 @@ func NewSpanWriter(options WriterOptions) *SpanWriter {
 	return writer
 }
 
-func (w *SpanWriter) writeIndexBatch(batchSpans []*Span) error {
+// Round-robin connection-selection strategy.
+func (w *SpanWriter) selectConn() clickhouse.Conn {
+	if len(w.conns) > 1 {
+		w.conns = append(w.conns[1:], w.conns[0])
+	}
+	return w.conns[0]
+}
```

Comment:
(and is there a way I can centralize/dedupe `Writer.selectConn` and `Factory.selectConn`, since they're identical?)